ansible/roles/nessus/files/nessus_base.py

```diff
@@ -80,19 +80,24 @@ def __make_request(self, target, method, payload=None):
             if self.token:
                 headers['X-Cookie'] = 'token={!s}'.format(self.token)
 
-            if target == FILE_UPLOAD:
-                headers = {}
-                headers['X-Cookie'] = 'token={!s}'.format(self.token)
-                files = {'Filedata': (BASE_POLICY_FILE_NAME, open(BASE_POLICY_FILE_NAME, 'rb'), 'text/xml')}
-                response = requests.post(self.url + target, headers=headers, files=files, verify=VERIFY_SSL)
-                LOGGER.info('Policy Uploaded to Nessus Server')
-                resp_dict = json.loads(response.text)
-                response = requests.post(self.url + POLICY_IMPORT, headers=headers, data={'file':resp_dict['fileuploaded']}, verify=VERIFY_SSL)
-                LOGGER.info('Base Policy Imported to Nessus Server Policies')
-            elif method == 'GET':
+            # if target == FILE_UPLOAD:
```

Comment:
Remove this stuff if it's not needed.  There's no need to leave dead code around that can be retrieved from the git history.